### DISCUSSION TITLE
[p222] 코드 질문

### DISCUSSION THREAD

**Discussion by gihohpkl12** [fsoftwareengineer/todo-application#79] [질문과 답변] [answered]
app-config.js에서 
export const API_BASE_URL = '{backedHost}'으로 되어있는데
이렇게 쓸 경우 VM185:1 Uncaught (in promise) SyntaxError: Unexpected token < in JSON at position 0 에러가 납니다.
이 부분을 export const API_BASE_URL = backedHost 이렇게 할 경우에 에러가 발생하지 않습니다.

**Comment by MJJ32** [comment, marked as answer]
'{backedHost}'  이부분 작은 따옴표를 써서 그럴수도 있습니다. 저도 그래서 한참 찾았어요 
' -> `  (왼쪽 탭키 위에있는 키)

**Reply by gihohpkl12** [reply]
이게 맞네요 감사합니다.
